### ISSUE TITLE
Selecting a part of a line

### ISSUE THREAD

**Issue by korayal** [nickbnf/glogg#82]
When we need to copy only a portion of a line, we always have to select that line, copy it to a text editor, then select the section we need and copy again.

I would be nice to have something like a method that we can double click the line (which would convert that line to a textbox) and copy that part and press Esc to revert it back to whatever it is.

**Comment by fretep** [comment]
Hey korayal, I actually think you can select part of a line, but you need to click and drag, not single or double click. Give it a shot and see how you go.

Regards, Peter

**Comment by korayal** [comment]
Yep, that one works :) sorry for that.

Well then to give this thread some purpose, we can replace this request for a change to be able to select multiple lines with CTRL+Click

**Comment by nickbnf** [comment]
You can select several consecutive lines using SHIFT+Click, but several non-consecutive line selections are not supported. It wouldn't be complicated to add it, but is it really a useful use case?

**Comment by korayal** [comment]
Yes, it is one of things that I try to do without noticing that it is not working :) it is useful when you make a filter, and then visually select (via quickly checking text in lines above or below them) among those filtered lines.
